app/services/idv/financials_step.rb

```diff
@@ -13,6 +13,10 @@ def submit
       result
     end
 
+    def vendor_invalid?
```

Comment:
rename this to `vendor_validation_failed?` It's the validation that failed, not the vendor who is invalid 😄